Resources/public/js/views/services/ez-notificationhubviewservice.js

```diff
@@ -19,13 +19,13 @@ YUI.add('ez-notificationhubviewservice', function (Y) {
      * @constructor
      * @extends eZ.TemplateBasedView
      */
-    Y.eZ.NotificationHubViewService = Y.Base.create('notificationHubViewService', Y.eZ.ViewService, [], {
+    Y.eZ.NotificationHubViewService = Y.Base.create('notificationHubViewService', Y.eZ.ViewService, [Y.eZ.SideViewService], {
         initializer: function () {
-            this.after('configChange', function () {
-                var config = this.get('config');
+            this.after('parametersChange', function () {
+                var parameters = this.get('parameters');
```

Comment:
because it does not mean the same thing, you might not have the same value for instance if there's a `getter` function on the attribute (and really the perf difference does not worth to bother)